### ISSUE TITLE
Google Api Error: forbidden: Cannot update a published APK.

### ISSUE THREAD

**Issue by ganeshub** [fastlane/fastlane#16922]
In my flutter project when i try to upload new apk to play store i get this error

```
....
....
....
[01:57:18]: Preparing apk at path '../build/app/outputs/apk/release/app-release.apk' for upload...
+-------------------------------------+----------------+
|                     Lane Context                     |
+-------------------------------------+----------------+
| DEFAULT_PLATFORM                    | android        |
| PLATFORM_NAME                       | android        |
| LANE_NAME                           | android deploy |
| GRADLE_ALL_APK_OUTPUT_PATHS         | []             |
| GRADLE_ALL_AAB_OUTPUT_PATHS         | []             |
| GRADLE_ALL_OUTPUT_JSON_OUTPUT_PATHS | []             |
| GRADLE_ALL_MAPPING_TXT_OUTPUT_PATHS | []             |
+-------------------------------------+----------------+
[01:58:43]: Google Api Error: forbidden: Cannot update a published APK. - Cannot update a published APK.

+------+-----------------------+-------------+
|              fastlane summary              |
+------+-----------------------+-------------+
| Step | Action                | Time (in s) |
+------+-----------------------+-------------+
| 1    | default_platform      | 0           |
| 2    | clean assembleRelease | 50          |
| 💥   | upload_to_play_store  | 87          |
+------+-----------------------+-------------+

[01:58:43]: fastlane finished with errors

[!] Google Api Error: forbidden: Cannot update a published APK. - Cannot update a published APK.

**Comment by dandre-hound** [comment]
I'm seeing the same issue. Any luck?

**Comment by ganeshub** [comment]
@dandre-hound Add new release and upload apk manually for the first time. After that fastlane works.
Still, I am not sure is this the right way.

**Comment by lorenamelor** [comment]
I have the same problem. Any solution?

**Comment by dandre-hound** [comment]
@lorenamelor I was unknowingly uploading an apk with a build number that had already been published. I just incremented the version code (using [fastlane-plugin-versioning_android](https://github.com/beplus/fastlane-plugin-versioning_android)) and the error went away. Hopefully this helps you!

**Comment by lorenamelor** [comment]
Thanks for the answer @dandre-hound , but I didn't understand the problem.
What build number do you refer to? Can you contextualize me a little more, please?

**Comment by dandre-hound** [comment]
Sure. By the way I actually meant version code, not build number. Your version code should be defined in app/build.gradle as `versionCode`. Try incrementing this number by 1 and uploading again. I had already uploaded an android apk file (the bundled application) with a version code of 716 when I ran into this error. Once I set the `versionCode` to 717, I was able to upload successfully. I only brought up fastlane-plugin-versioning_android because it automatically increments this number for you whenever you run fastlane. Does this help?
